pkg/service/restore/restore_integration_test.go

```diff
@@ -334,3 +338,119 @@ func TestRestoreTablesVnodeToTabletsIntegration(t *testing.T) {
 
 	validateTableContent[int, int](t, h.srcCluster.rootSession, h.dstCluster.rootSession, ks, tab, c1, c2)
 }
+
+func TestRestoreTablesPausedIntegration(t *testing.T) {
```

Comment:
nice !
I copied this test together with helpers to master branch, expecting that it is gonna fail with the same error that was met on prod env and it did :)
```
    restore_integration_test.go:448: create dir worker: get table version: not found
```